shell/app/node_main.cc

```diff
@@ -165,7 +184,15 @@ int NodeMain(int argc, char* argv[]) {
       versions.SetReadOnly(ELECTRON_PROJECT_NAME, ELECTRON_VERSION_STRING);
     }
 
-    node::LoadEnvironment(env);
+    // TODO(codebytere): we should try to handle this upstream.
+    {
+      node::InternalCallbackScope callback_scope(
+          env, v8::Local<v8::Object>(), {1, 0},
+          node::InternalCallbackScope::kAllowEmptyResource |
+              node::InternalCallbackScope::kSkipAsyncHooks);
```

Comment:
Maybe add a `DCHECK(!callback_scope.Failed())` here?

:+1: on trying to handle this upstream

Reply:
I don't think we'll need this - i matched upstream and it looks like those externalities aren't a concern there.